src/elements/play-pen-header.ts

```diff
@@ -149,14 +202,14 @@ export class PlayPenHeader extends LitElement {
             label="Docs"
             @click=${() => openURL('https://developers.reddit.com/docs')}
           ></play-button
-          ><play-button
-            appearance="bordered"
+          ><play-project-button
             size="small"
-            icon="download-outline"
-            title="Export Pen"
-            label="Export"
-            @click=${() => this._export.open()}
-          ></play-button
+            .srcByLabel=${this.srcByLabel}
+            @open-export-dialog=${() => this._export.open()}
+            @save-project=${() => this.openSaveDialog()}
```

Comment:
:no_bell: fat arrow is optional here due to [automatic this binding](https://lit.dev/docs/components/events/#understanding-this-in-event-listeners).